frontend/homepage/src/components/Home_p1/Navbar/CreateAccount/CreateAccount.js

```diff
@@ -34,29 +30,47 @@ function CreateAccount(props) {
             <input type="password" name="pwd" />
           </label>
           <br />
+          {isOwner&& <div>
           <label>
             Role:
-            <select onChange={IsOwner} required name="role">
+            
+            <select required name="role">
               <option defaultValue="None">None</option>
               <option value="GOVTOff">Owner</option>
```

Comment:
Please comment out this line as we are not allowing the owner to create more owner accounts.

Reply:
the changes are now made, have a look.

Reply:
Looks good, merging!